src/strategies_and_params/shiftstrategy.jl

```diff
@@ -9,27 +10,76 @@ Passed as a parameter to [`lomc!`](@ref).
 * [`LogUpdate`](@ref)
 * [`LogUpdateAfterTargetWalkers`](@ref) - FCIQMC standard
 * [`DoubleLogUpdateAfterTargetWalkers`](@ref)
+
+# Extended help
+
+Internally
+To implement a custom strategy, define a new subtype of `Rimu.ShiftStrategy` and implement
+methods for:
+- [`Rimu.update_shift_parameters!`](@ref) - to update the `shift_parameters`
+- [`Rimu.initialise_shift_parameters`](@ref) - (optional) to initialise and construct a
+    custom implementation of the `shift_parameters`. The default implementation is
+    [`Rimu.DefaultShiftParameters`](@ref).
 """
 abstract type ShiftStrategy end
 
 """
-    update_shift(s <: ShiftStrategy, shift, shiftMode, tnorm, pnorm, dτ, step, df, v_new, v_old)
-Update the shift according to strategy `s`. See [`ShiftStrategy`](@ref).
+    DefaultShiftParameters
+Default mutable struct for storing the shift parameters.
+
+See [`ShiftStrategy`](@ref), [`initialise_shift_parameters`](@ref).
 """
-update_shift
+mutable struct DefaultShiftParameters{S, N}
+    shift::S # for current time step
+    pnorm::N # norm from previous time step
+    time_step::Float64
+    counter::Int
+    shift_mode::Bool
```

Comment:
Is the counter necessary? Should `shift_mode` be a part of a separate type of shift parameters struct?

Reply:
Maybe this is not elegant, but this way all existing `ShiftStrategy`s can use this struct.